src/Gaufrette/Adapter/GoogleCloudStorage.php

```diff
@@ -351,4 +362,30 @@ private function getObjectData($path, $options = array())
             return false;
         }
     }
+
+    /**
+     * @param $filename
+     * @param null $expires
+     * @return string
+     * @throws \Google_Auth_Exception
+     */
```

Comment:
Can you document this method above, and also provide the correct types for the two params.
I would have thought also that you could have just returned the `selfLink` property from the `getObjectData` method, especially if using a public acl.